libcst/codegen/gen_visitor_functions.py

```diff
@@ -0,0 +1,181 @@
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+# pyre-strict
+import inspect
+from collections import defaultdict
+from dataclasses import fields
+from typing import Dict, Generator, List, Mapping, Sequence, Set, Type
+
+import libcst as cst
+
+
+DO_NOT_CARE_TYPES: Set[str] = {"CSTNode", "ABC", "object"}
```

Comment:
Optional: You could check `issubclass(cls, CSTNode) and not isinstance(cls, CSTNode)` instead. It looks like you're already doing `issubclass(node, cst.CSTNode)` in `_get_nodes`.

Reply:
I do that because I'm walking the tree until we get to nodes that are direct descendants of CSTNode, not to exclude nodes. I could issubclass() as well I guess.